UI/src/components/Home.vue

```diff
@@ -18,44 +17,59 @@
             </div>
             <div class="welcomed w2">
                 <h2>{{ $t(`Community resources`) }}</h2>
-                <p>{{ $t(`Community support for LedgerSMB is a available through various
-                  channels, ranging from static all the way to near-realtime (depending
-                  on availability of people):`) }}</p>
+                <p>{{ $t(`i18n.home.communitySupport`) }}</p>
```

Comment:
Why are you using backticks in some cases and double quotes in others?

Reply:
Fixed